### PR TITLE
fix typing

### PR DESCRIPTION
**Pull request by fraimondo**
* [x] fix typing
* [ ] description of feature/fix
* [x] tests added/passed
* [ ] add an entry to the [latest changes](../docs/changes/latest.inc)

### CODE REVIEW

File: junifer/storage/sqlite.py [hunk 1]
```diff
@@ -112,18 +109,19 @@ def get_engine(
...
+        uri = (
+            f"sqlite:///{self.uri.parent}/"  # type: ignore
+            f"{prefix}{self.uri.name}"  # type: ignore
+        )
         return create_engine(uri, echo=False)
 
     def _save_upsert(
         self,
-        df: pd.DataFrame,
+        df: Union[pd.DataFrame, pd.Series],
```
Comment (synchon) [inline junifer/storage/sqlite.py:124]: The docstring needs to be updated.

File: junifer/storage/sqlite.py [hunk 2]
```diff
@@ -385,7 +385,7 @@ def store_metadata(self, meta: Dict) -> str:
             self._save_upsert(meta_df, "meta", engine)
         return f"meta_{meta_md5}"
 
-    def store_df(self, df: pd.DataFrame, meta: Dict) -> None:
+    def store_df(self, df: Union[pd.DataFrame, pd.Series], meta: Dict) -> None:
```
Comment (synchon) [inline junifer/storage/sqlite.py:388]: The docstring needs to be updated.

File: junifer/storage/sqlite.py [hunk 3]
```diff
@@ -445,7 +445,7 @@ def store_matrix(
 
         Parameters
         ----------
-        data : dict
+        data : numpy.ndarray
```
Comment (synchon) [inline junifer/storage/sqlite.py:448]: `ArrayLike`?

**Review by synchon — CHANGES_REQUESTED** [review]

File: junifer/storage/sqlite.py (hunk 3 above)
Comment (fraimondo) [reply junifer/storage/sqlite.py:448]: does not work with ArrayLike because we use `.shape`

**Review by synchon — APPROVED** [review]

**Pull request merged**